src/test/java/api/loans/CheckOutByBarcodeFixedDueDateScenariosTest.java

```diff
@@ -64,7 +64,8 @@ public void shouldUseMoveToThePreviousOpenDayStrategyForLongTermLoanPolicyWhenDu
     UUID loanPolicyId = loanPolicyClient.create(loanPolicy).getId();
     UUID requestPolicyId = requestPoliciesFixture.allowAllRequestPolicy().getId();
     UUID noticePolicyId = noticePoliciesFixture.activeNotice().getId();
-    useLoanPolicyAsFallback(loanPolicyId, requestPolicyId, noticePolicyId);
+    UUID overdueFinePolicyId = overdueFinePoliciesFixture.facultyStandard().getId();
+    useLoanPolicyAsFallback(loanPolicyId, requestPolicyId, noticePolicyId, overdueFinePolicyId);
```

Comment:
This code exists 3 times. Can you extract it into a method? Example:

    private void use(LoanPolicyBuilder loanPolicy) {
      UUID loanPolicyId = loanPolicyClient.create(loanPolicy).getId();
      UUID requestPolicyId = requestPoliciesFixture.allowAllRequestPolicy().getId();
      UUID noticePolicyId = noticePoliciesFixture.activeNotice().getId();
      UUID overdueFinePolicyId = overdueFinePoliciesFixture.facultyStandard().getId();
      useLoanPolicyAsFallback(loanPolicyId, requestPolicyId, noticePolicyId, overdueFinePolicyId);
    }

Reply:
@julianladisch I got your point. Thank you!